crates/build/src/lib.rs

```diff
@@ -288,6 +288,7 @@ fn exec_cargo_for_wasm_target(
         Workspace::new(&crate_metadata.cargo_meta, &crate_metadata.root_package.id)?
             .with_root_package_manifest(|manifest| {
                 manifest
+                    .with_added_crate_type("cdylib")?
                     .with_removed_crate_type("rlib")?
```

Comment:
I think what we want to express is that we want to set it to only `cdylib` and remove everything else.